### ISSUE TITLE
Setting tags-from-host=true overrides other specified tags

### ISSUE THREAD

**Issue by nicklaw5** [buildkite/agent#967]
As far as I can tell this has only recently stopped working. Previously I believe it was possible to merge both `tags-from-host` and `tags`.

Below is the configuration file I'm using:

```yaml
# The token from your Buildkite "Agents" page
token="<redacted>"

# The name of the agent
name="%hostname-%n"

# Tags for the agent (default is "queue=default")
tags="queue=gjg-default,foo=bar"

# Include the host's meta-data as tags (hostname, machine-id, and os)
tags-from-host=true

# Path to where the builds will run from
build-path="/var/lib/buildkite-agent/builds"

# Directory where the hook scripts are found
hooks-path="/etc/buildkite-agent/hooks"

# When plugins are installed they will be saved to this path
plugins-path="/etc/buildkite-agent/plugins"
```

With this configuration only the host tags are attached to the agent:

![image](https://user-images.githubusercontent.com/7908386/55357597-16687a80-5519-11e9-8974-dd68947842e7.png)

However, if set `tags-from-host=false` my specific tags are attached to the agent:

![image](https://user-images.githubusercontent.com/7908386/55357426-b7a30100-5518-11e9-98f4-cf4da3777d5d.png)

Is this intended behaviour? Or should both host and specified tags be attached the agent?

**Comment by lox** [comment]
We're investigating, looks like a regression in 3.10.2.

**Comment by lox** [comment]
We have a fix in https://github.com/buildkite/agent/issues/967, will get it released shortly.

**Comment by nicklaw5** [comment]
Thanks @lox for the prompt response 👍

**Comment by lox** [comment]
No problems, apologies for the breakage in a point release!

**Comment by nicklaw5** [comment]
Don't be so hard on yourself, we've all been there :)

**Comment by lox** [comment]
Fixed in v3.10.3
